view/utils/init-table-sorting-on-client.js

```diff
@@ -1,12 +1,9 @@
 'use strict';
 
-module.exports = function (jQuerySelector) {
-	if (!window.jQuery) return;
-	var checkExist = setInterval(function () {
-		var element = window.jQuery(jQuerySelector);
-		if (element.length) {
-			element.tablesorter();
-			clearInterval(checkExist);
-		}
+module.exports = function (domElement, tableSorterOpts) {
+	setInterval(function () {
+		var element = window.jQuery(domElement);
```

Comment:
This one is still not addressed. We need sth like:
`if (!window.jQuery) return;`

before we even call `setInterval`